targets/TARGET_NUVOTON/TARGET_M451/serial_api.c

```diff
@@ -192,51 +196,61 @@ void serial_init(serial_t *obj, PinName tx, PinName rx)
 #endif
 
     // For stdio management
-    if (obj == &stdio_uart) {
+    if (obj->serial.uart == STDIO_UART) {
         stdio_uart_inited = 1;
-        /* NOTE: Not required anymore because stdio_uart will be manually initialized in mbed-drivers/source/retarget.cpp from mbed beta */
-        //memcpy(&stdio_uart, obj, sizeof(serial_t));
+        memcpy(&stdio_uart, obj, sizeof(serial_t));
```

Comment:
@0xc0170 That was to meet mbed OS 3.0. Now I change it back to meet mbed OS 2.0/5.0.